### PR TITLE
 Run examples, small corrections

### PR DESCRIPTION
**Pull request by oligus**
* If no autoload found in parent, use local autoload
* Example 2, explode() expects parameter 2 to be string, array given, fixed
* Example 2, Unknown argument "truncated" on field "title", fixed

### CODE REVIEW

File: examples/02_blog/index.php [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
 
 $processor = new Processor($schema);
-$payload = 'mutation { likePost(id:5) { title(truncated: false), status, likeCount } }';
-$payload = '{ latestPost { title, status, likeCount } }';
-$payload = '{ pageContentUnion { ... on Post { title, summary } ... on Banner { title, imageLink } } }';
-$payload = '{ pageContentInterface { title} }';
+// $payload = 'mutation { likePost(id:5) { title(truncated: false), status, likeCount } }';
+// $payload = '{ latestPost { title, status, likeCount } }';
+// $payload = '{ pageContentUnion { ... on Post { title, summary } ... on Banner { title, imageLink } } }';
+// $payload = '{ pageContentInterface { title} }';
```
Comment (Jalle19) [inline examples/02_blog/index.php]: Why did you comment these out?

Comment (oligus) [reply examples/02_blog/index.php]: Honestly, I cant recall

Comment (Jalle19) [reply examples/02_blog/index.php]: Can you uncomment them so we can get this in?

Comment (oligus) [reply examples/02_blog/index.php]: Now I recall, this is just examples, you can just run it with `php index.php`or similar. The $payload variable gets overwritten by the next line immediately, so you run just one at a time **anyway** :-)

**Pull request merged**